infra/app/env-template/main.tf

```diff
@@ -269,6 +281,10 @@ module "analytics" {
       name  = "MATOMO_DATABASE_DBNAME",
       value = local.analytics_database_name,
     },
+    {
+      name  = "MATOMO_URL_BASE",
+      value = var.analytics_url,
+    }
```

Comment:
I don't think matomo itself needs its own url?

Reply:
The URL base is what's used to help build the matomo calls.
e.g. in dev this would be something like:
`dev-analytics.wic-services.org/matomo.php` -> allows the dev environment to make a call to matomo

Reply:
But that's for the participant container. The analytics container doesn't accept this env var